pkg/k8s/watchers/cilium_envoy_config_test.go

```diff
@@ -56,9 +54,9 @@ spec:
 func (s *K8sWatcherSuite) TestParseEnvoySpec(c *C) {
 	jsonBytes, err := yaml.YAMLToJSON([]byte(envoySpec))
 	c.Assert(err, IsNil)
-	var buf bytes.Buffer
-	json.Indent(&buf, jsonBytes, "", "\t")
-	fmt.Printf("JSON spec:\n%s\n", buf.String())
+	//var buf bytes.Buffer
+	//json.Indent(&buf, jsonBytes, "", "\t")
+	//fmt.Printf("JSON spec:\n%s\n", buf.String())
```

Comment:
Why not just remove the code?

Reply:
I'd guess these are meant for debugging, so let's just remove them and file an issue to clean up the rest as a followup.